src/devices_models/device_constructors/hydrogeneration_constructor.jl

```diff
@@ -21,7 +21,9 @@ function construct_device!(
 
     #Constraints
     activepower_constraints!(psi_container, devices, model, S, get_feedforward(model))
```

Comment:
This constraints can't be eliminated for this model. Hydro doesn't require constant pf assumptions. For the HydroDispatchFormulations it should add reactive power bounds.

Reply:
I think @claytonpbarrows  had suggested I do that, I also if we want a reactive power constraints then we need to define a new constraint that works with the data in Hydro, as its different than Thermal if I'm correct

Reply:
Reactive power in Hydro is the same as in Thermal